src/rust/engine/process_executor/.gitignore

```diff
@@ -0,0 +1 @@
+Cargo.lock
```

Comment:
Can you add a comment indicating why this is safe?

Presumably we always build these in the context of the parent project?